### PR TITLE
Imagemagick7 git

### PR DESCRIPTION
**Pull request by wandrewkeech**
Builds a mostly-working kitchen sink version of IM7 in `devmode` non-confinement

### CODE REVIEW

**Comment by dholbach** [comment]
Is `imagemagick-edge/imagemagick_7.0.2~git_source.tar.bz2` needed?

**Comment by wandrewkeech** [comment]
Ah, definitely not.

**Comment by dholbach** [comment]
It looks like you changed `imagemagick-stable/snapcraft.yaml` and didn't add a file in `imagemagick-edge`.

**Comment by wandrewkeech** [comment]
I'm obviously not quite qualified to use `git` unsupervised

**Comment by dholbach** [comment]
You're a bit hard on yourself. You're doing great work!

File: imagemagick-edge/snapcraft.yaml [hunk 1]
```diff
@@ -0,0 +1,102 @@
+name: imagemagick-stable
```
Comment (dholbach) [inline imagemagick-edge/snapcraft.yaml]: Maybe this should be `-edge`?

File: imagemagick-stable/snapcraft.yaml [hunk 2]
```diff
@@ -1,5 +1,10 @@
+<<<<<<< HEAD:imagemagick-edge/snapcraft.yaml
+name: imagemagick-edge
+version: 7.0.2~git
+=======
 name: imagemagick-stable
 version: 6.9.7
+>>>>>>> ddb719ede8d95bd9e1362f9dd91033c1581ffb5c:imagemagick-stable/snapcraft.yaml
 summary: ImageMagick is a software suite to create, edit, or convert bitmap images.
```
Comment (dholbach) [inline imagemagick-stable/snapcraft.yaml]: Is this a conflict?

Comment (wandrewkeech) [inline imagemagick-stable/snapcraft.yaml]: You've uncovered the stuff I was rushing through. Made some errors `git rm`ing, and just `rm`ed and messing up tidying

File: imagemagick-stable/snapcraft.yaml [hunk 3]
```diff
@@ -56,11 +56,19 @@ parts:
             - --enable-shared=yes
             - --enable-static=yes
             - --with-autotrace=yes
+<<<<<<< HEAD:imagemagick-edge/snapcraft.yaml
+            - --with-dps=no
+            - --with-dps=yes
+            - --with-flif=no
+=======
+>>>>>>> ddb719ede8d95bd9e1362f9dd91033c1581ffb5c:imagemagick-stable/snapcraft.yaml
             - --with-fpx=no
```
Comment (dholbach) [inline imagemagick-stable/snapcraft.yaml]:
```
daniel@daydream:/tmp/snappy-playpen3/imagemagick-stable$ snapcraft cleanbuild
Issues while validating snapcraft.yaml: could not found expected ':' on line 62 of snapcraft.yaml
daniel@daydream:/tmp/snappy-playpen3/imagemagick-stable$ 
```

File: README.md [hunk 4]
```diff
@@ -17,10 +17,12 @@ This project currently includes the following snaps:
...
 | :white_check_mark:  | `galculator`       |                           |
 | :red_circle:        | `heroku`           |                           |
-| :red_circle:        | `imagemagick6-stable`|                         |
+| :white_check_mark:  | `imagemagick6-stable`|                         |
+| :red_circle:        | `imagemagick7-git` |                           |
 | :white_check_mark:  | `keepassx`         | [keepassx-elopio][]       |
 | :white_check_mark:  | `kpcli`            | [kpcli-elopio][kpcli]     |
 | :white_check_mark:  | `leafpad`          |                           |
+| :red_circle:        | `mate-deskop`      |                           |
 | :white_check_mark:  | `minetest`         |                           |
```
Comment (dholbach) [inline README.md]: I guess this shouldn't be there.

**Comment by dholbach** [comment]
Nice work! :)

**Pull request merged**